### ISSUE TITLE
use of CU logo

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/projectile-motion#142]
I ran across this scenario quite by accident:

![screenshot_353](https://user-images.githubusercontent.com/3046552/29332272-b2ebce48-81bc-11e7-8452-933d8a66a50b.png)

CU's brand policy is at http://www.cu.edu/ope/aps/2025.  
Approved logo variations are at http://www.colorado.edu/brand/logo/logo-variations. 
Approved colors are at http://www.colorado.edu/brand/visual-identity/typography-color.  

In a nutshell, the logo in above snapshot deviates from the CU brand, "Entities of the University may not create their own logo and must follow university branding standards", and "Any request to deviate from the brand identity standards must be submitted in writing to the University Brand Identity Standards Board."

**Comment by arouinfar** [comment]
Thanks @pixelzoom. I've updated assets to include the appropriate CU logo, and have emailed them @andrea-phet.

**Comment by andrea-phet** [comment]
Incorporated. Assigning to @arouinfar to review and close.

**Comment by arouinfar** [comment]
👍

**Comment by pixelzoom** [comment]
@arouinfar are you sure about this? It almost doesn't look like a flag anymore, and the CU logo is barely legible on my MacBook Pro.

Before:
![screenshot_354](https://user-images.githubusercontent.com/3046552/29337963-a4df5540-81d0-11e7-8e8e-f84ff7bde835.png)

After:
![screenshot_355](https://user-images.githubusercontent.com/3046552/29337968-a8e1c0e2-81d0-11e7-8377-aa5c291a2918.png)

**Comment by arouinfar** [comment]
I would prefer to use just the interlocking CU logo, but that seems to be reserved for athletics. @ariel-phet and I worked on the artwork together, and he approved, but should probably take a look in context.

**Comment by pixelzoom** [comment]
Maybe a Colorado flag would hold up better, and provide the same "easter egg" factor?

![screenshot_356](https://user-images.githubusercontent.com/3046552/29338089-08f4a512-81d1-11e7-81dd-98750b992004.png)

**Comment by arouinfar** [comment]
From @ariel-phet:

> We're going to go with the interlocking CU logo (from the branding site), but without the "University of Colorado Boulder". If it comes up, we'll be asking for forgiveness, not permission. 

I've emailed the asset to @andrea-phet for integration.

![flatirons](https://user-images.githubusercontent.com/8419308/29340095-97834190-81da-11e7-8e33-674da490c2cc.png)

**Comment by arouinfar** [comment]
Looks good!
